client/payment-details/summary/index.tsx

```diff
@@ -246,10 +267,16 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 						</p>
 						<div className="payment-details-summary__breakdown">
 							{ renderStorePrice ? (
-								<p>
+								<p
+									className="payment-details-summary__breakdown__original-amount"
```

Comment:
I'm thinking of this as "what the shopper paid" or shopper currency.

Reply:
Great idea 💡 – I've adjusted classNames and aria-labels to indicate "Payment amount in shopper's currency" instead of "Payment amount in original amount": cea9679.